runtime/worker_bootstrap.rs

```diff
@@ -47,6 +48,7 @@ impl Default for BootstrapOptions {
       enable_testing_features: Default::default(),
       debug_flag: Default::default(),
       ts_version: Default::default(),
+      locale: "en-EN".to_string(),
```

Comment:
this seems like an error - there's no region `EN`.

Reply:
Thanks! Silly mistake on my part, I'll fix it